docs/guided-remediation.md

```diff
@@ -224,10 +224,24 @@ A vulnerability is only considered if it satisfies all the conditions set by the
 
 ### Dependency upgrade options
 
-The following flags may be used to limit the patches allowed for your dependencies:
+The following flag may be used to limit the patches allowed for your dependencies:
 
-- `--disallow-major-upgrades`: Do no allow patches that would result in the major version number of any dependency from being changed.
-- `--disallow-package-upgrades=<comma-separated list of package names>`: Do no allow patches to any of the listed packages.
+- `--upgrade-config=<[package-name:]level>` Sets the maximum upgrade level allowed for a package. Can be repeated for multiple packages.
+
+  `level` is the SemVer component to allow updates to, can be one of `major`, `minor`, `patch`, or `none`. e.g. If a package was at version `1.2.3`
+
+  - `major` allows for updates to any version `>=1.2.3`
+  - `minor` allows for updates `>=1.2.3, <2.0.0`
+  - `patch` allows for updates `>=1.2.3, <1.3.0`
+  - `none` disallows any updates
+
+  If `package-name:` is omitted, `level` is applied to all packages. Default is `--upgrade-config=major`.
```

Comment:
Can we add some detail to describe precedence? 

i.e. our behavior is to take the most specific setting right? if a specific package is specified in a config, that takes precedence over a global config.

Reply:
+1 - it is not clear to me whether the specific settings overwrite the global ones until I see the example usage.

Reply:
Added some explanation w.r.t precedence here